example/example.cpp

```diff
@@ -54,6 +54,30 @@ int main() {
     assert(status == LineairDB::TxStatus::Committed);
   }
 
+  {
+    LineairDB::Database db;
+    LineairDB::TxStatus status;
+
+    // Delete API example: tombstone remains even if we read again in same tx
```

Comment:
> tombstone remains even if we read again in same tx

What is the meaning of it?
Comments delving into implementation details should not be written in examples (because while interfaces are immutable, implementations can change).